notification-portlet-webapp/src/main/webapp/less/modal.less

```diff
@@ -0,0 +1,94 @@
+.notification-flat {
+  color: #0074b7;
+  transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
+  min-height: 36px;
+  box-sizing: border-box;
+  align-items: center;
+  line-height: 1em;
+  font-family: Arimo;
+  font-weight: 500;
+  border-radius: 2px;
+  text-transform: uppercase;
+  justify-content: center;
+  background-color: transparent;
+  padding: 11px 16px;
+  display: inline-flex;
+  outline: none;
+  border: 0;
+}
+
+.notification-flat:hover {
+  background-color: rgba(0, 116, 183, 0.12);
+}
+
+.notification-modal-mask {
+  position: fixed;
+  z-index: 9998;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, .5);
+  display: table;
+  transition: opacity .3s ease;
+}
+
+.notification-modal-wrapper {
+  vertical-align: middle;
+  width: 60%;
+  margin: 30px auto;
+}
+
+.notification-modal-container {
+  width: 300px;
+  margin: 0px auto;
+  padding: 20px 30px;
+  background-color: #fff;
+  border-radius: 2px;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, .33);
+  transition: all .3s ease;
+}
+
+.notification-modal-header {
+  margin-top: 0;
+  padding: 24px 24px 24px 24px;
+  background-color: #4065B0;
+}
+
+.notification-modal-body {
+  flex: 1 1 auto;
+  padding: 24px 24px 24px 24px;
+  overflow-y: auto;
+}
+
+.notification-modal-footer {
+  flex: 0 0 auto;
+  margin: 8px 4px;
+  border: none;
+  padding: 0px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.notification-modal-enter {
+  opacity: 0;
+}
+
+.notification-modal-leave-active {
+  opacity: 0;
+}
+
+.notification-modal-content {
+  webkit-box-shadow: 0 5px 15px rgba(0, 0, 0, 0.5);
+  box-shadow: 0 5px 15px rgba(0, 0, 0, 0.5);
+  background-color: #ffff;
```

Comment:
:question: is `rgba` hex notation intended here? https://caniuse.com/#search=hex